src/audio/kpb.c

```diff
@@ -784,12 +797,21 @@ static int kpb_params(struct comp_dev *dev,
 static int kpb_prepare(struct comp_dev *dev)
 {
 	struct comp_data *kpb = comp_get_drvdata(dev);
+	struct sof_ipc_stream_params params;
 	int ret = 0;
 	int i;
 	size_t hb_size_req = KPB_MAX_BUFFER_SIZE(kpb->config.sampling_width, kpb->config.channels);
 
 	comp_dbg(dev, "kpb_prepare()");
 
+	kpb_set_params(dev, &params);
+
+	ret = kbp_verify_params(dev, &params);
+	if (ret < 0) {
+		comp_err(dev, "kpb_prepare: pcm params verification failed");
```

Comment:
I'd propose not to use function names at least in new logging prints - even if they're otherwise used throughout the code. But in fact see `comp_cl_err()` call 6 lines down - it has no function name in it. Zephyr adds function names by itself. I'd also not use function names in added prints below - in module_adapter.c etc., even if so far they're used there.